src/Repository/CaseEntityRepository.php

```diff
@@ -18,4 +19,15 @@ public function __construct(ManagerRegistry $registry)
     {
         parent::__construct($registry, CaseEntity::class);
     }
+
+    public function findLatestCaseByMunicipality(Municipality $municipality): ?CaseEntity
+    {
+        return $this->createQueryBuilder('c')
+            ->where('c.municipality = :municipality')
+            ->setParameter('municipality', $municipality->getId()->toBinary())
+            ->orderBy('c.createdAt', 'DESC')
+            ->setMaxResults(1)
```

Comment:
Tiny thing. Is this necessary when you are calling getOneOrNullResult as the last method?